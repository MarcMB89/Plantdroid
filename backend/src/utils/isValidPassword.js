function isValidPassword(password) {
  return password === this.password;
}
module.exports = { isValidPassword };
