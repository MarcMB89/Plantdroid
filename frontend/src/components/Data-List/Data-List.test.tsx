import React from 'react';
import {render} from '../../utils/test.utils';
import DataList from './Data-List';

let navigation: any = {
  navigate: jest.fn(),
};

describe('Given a DataList Component', () => {
  describe('When it is rendered', () => {
    let screen: any;
    beforeEach(() => {
      screen = render(<DataList navigation={navigation} />);
    });
    test('Then screen should match the snapshot', () => {
      expect(screen).toMatchSnapshot();
    });
  });
});

navigation = {navigation};
