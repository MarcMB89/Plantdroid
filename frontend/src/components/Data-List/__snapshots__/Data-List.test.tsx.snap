// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Given a DataList Component When it is rendered Then screen should match the snapshot 1`] = `
<RCTSafeAreaView
  emulateUnlessSupported={true}
  style={
    Object {
      "flex": 1,
    }
  }
>
  <View
    style={
      Object {
        "backgroundColor": "#dcedc8",
        "borderColor": "#212121",
        "flex": 1,
      }
    }
  >
    <View>
      <RCTScrollView
        data={Array []}
        getItem={[Function]}
        getItemCount={[Function]}
        keyExtractor={[Function]}
        onContentSizeChange={[Function]}
        onLayout={[Function]}
        onMomentumScrollBegin={[Function]}
        onMomentumScrollEnd={[Function]}
        onScroll={[Function]}
        onScrollBeginDrag={[Function]}
        onScrollEndDrag={[Function]}
        removeClippedSubviews={false}
        renderItem={[Function]}
        scrollEventThrottle={50}
        stickyHeaderIndices={Array []}
        viewabilityConfigCallbackPairs={Array []}
      >
        <View />
      </RCTScrollView>
    </View>
  </View>
</RCTSafeAreaView>
`;
